week-2/square/square.go

```diff
@@ -0,0 +1,27 @@
+package square
+
+type Point struct {
+	x, y int
+}
+
+type Square struct {
+	start Point
+	a     uint
+}
+
+func New(x int, y int, a uint) Square {
+	return Square{Point{x, y}, a}
+}
+
+func (s Square) End() Point {
+	endPoint := Point{s.start.x + int(s.a), s.start.y - int(s.a)}
```

Comment:
Variable 'endPoint' is redundant, you can just write `return Point{s.start.x + int(s.a), s.start.y - int(s.a)}`